header/headertest/testing.go

```diff
@@ -44,6 +43,10 @@ type TestSuite struct {
 	head *header.ExtendedHeader
 }
 
+func NewStore(t *testing.T) libhead.Store[*header.ExtendedHeader] {
+	return headertest.NewStore[*header.ExtendedHeader](t, NewTestSuite(t, 3), 10)
```

Comment:
do we want to allow these vals to be passed as params here?

Reply:
In this case, not. All our tests do not care how many validators are there, as well as how many headers we pregenerate

Reply:
If such granularity is needed, it is still possible